packages/manager/cypress/e2e/core/longview/longview.spec.ts

```diff
@@ -154,15 +127,6 @@ describe('longview', () => {
           cy.findByText('Waiting for data...');
         });
 
```

Comment:
```suggestion
      // Install Longview on Linode by SSHing into machine and executing cURL command.
      installLongview(linodeIp, linodePassword, installCommand);
```
In this case we still want to call `installLongview`, we just don't want to log its output. (This is going to be moot soon thanks to M3-8107, but I'm not sure this test will pass reliably in the meantime without this)

Reply:
Thanks! I reverted the changes and put the ```installlongview()``` back.